src/bidiMapper/modules/context/NavigationTracker.spec.ts

```diff
@@ -182,18 +209,31 @@ describe('NavigationTracker', () => {
         (await navigation.finished).eventName,
         NavigationEventName.NavigationFailed,
       );
-      assert.equal(navigationTracker.currentNavigationId, initialNavigationId);
+      // Current navigation is the not yet finished `YET_ANOTHER_URL` navigation.
+      assert.notEqual(
```

Comment:
If the navigation to `YET_ANOTHER_URL` is not finished why is this assert.notEqual .
Which Id is expected here, from which navigation.

Reply:
the navigation to `YET_ANOTHER_URL` is the current one, but we don't have it's ID

Reply:
updated comment